frontend/pages/club/[club]/edit.js

```diff
@@ -312,14 +312,29 @@ class ClubForm extends Component {
   }
 
   render() {
-    const { authenticated, schools, majors, years, tags } = this.props
+    const { authenticated, userInfo, schools, majors, years, tags } = this.props
     const { club, invites, editMember } = this.state
```

Comment:
```suggestion
    const { club, invites, isEdit, editMember } = this.state
```
Probably need this as well